tests/bridge-marker_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Red Hat, Inc.
+// Copyright 2014 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("bridge-marker", func() {
+	Describe("pod availability tests", func() {
+		Context("pod availability tests", func() {
+			It("assert pods exists", func() {
+				isPodPresent := false
+				pods, _ := clientset.CoreV1().Pods("").List(v1.ListOptions{})
+				for _, pod := range pods.Items {
+					if len(pod.ObjectMeta.OwnerReferences) == 0 {
+						continue
+					}
+					owner := pod.ObjectMeta.OwnerReferences[0].Name
+					if owner == "bridge-marker" {
```

Comment:
it's unclear what's the goal of this test case. it doesn't check that a pod serves any function, not that the pod is up and running, that it reported the bridges, not even that it's running the correct image; just that a pod with a particular name is created. This doesn't seem like a particularly useful check. (And it will break if someone decides to rename their pod).

Reply:
The only goal is to check that the pod is added.
I can either change it to check the pod by name ("bridge-marker*"), or remove the test.

Reply:
The way it is written, I don't think it's of much help so I would suggest to kill it. You have other cases that are more involving and check functionality.

Reply:
Removed